packages/devextreme/js/ui/load_panel.js

```diff
@@ -136,6 +139,19 @@ const LoadPanel = Overlay.inherit({
         this.setAria(aria, this.$wrapper());
```

Comment:
I agree with using `_getAriaAttributes` here to reduct copypaste but seems like `this.setAria` should still work well

Reply:
> but seems like this.setAria should still work well

@ksercs 

We use 'aria-label' prop name in aria object when we get aria configuration. So, we should not use this.setAria(). This method will set 'aria-aria-label' attribute to element.

And we need to use 'aria-label' prop name because we set aria to loadIndicator by elementAttr option. Component will use element.attr() method to set attribute, so if we will use just 'label' prop name, we will get label attribute on element like this: label='value' instead of aria-label='value'.

Reply:
Sure, now I see, u're right